docs/platform-specification.md

```diff
@@ -1361,7 +1361,10 @@ OpenOCD server specific configurations:
 
 - `debug.server.openocd.path`: is the absolute path to the OpenOCD directory
 - `debug.server.openocd.scripts_dir`: is the absolute path to the OpenOCD scripts directory
-- `debug.server.openocd.scripts.N`: is a list of OpenOCD scripts to run (where N is a number starting from 0)
+- `debug.server.openocd.scripts.N`: is a list of OpenOCD scripts to run, where N is a number (a sequence of
+  non-consecutive numbers is allowed)
+- `debug.server.openocd.script`: if there is only one OpenOCD script to run, this directive con be used instead of the
```

Comment:
I would mention that is here only for backward compatibility (legacy stuff) and advise using `scripts.0` instead

Reply:
I'm wondering: since we are going to support it forever, it just adds cognitive load without any actual benefit.